packages/legend-application/style/components/shared/_packageable-element-format-option.scss

```diff
@@ -41,6 +41,21 @@
   }
 }
 
+.packageable-element-format-option-label-type {
```

Comment:
I'll review this later, I like the preview you have, but let's try out a few color combination.

Reply:
updated PR description with some combos

Reply:
I think this color scheme is fine. It's consistent with other spots. Thanks a lot for the different suggestions you had given!